src/storage/gcloud_storage.rs

```diff
@@ -179,6 +181,38 @@ pub fn upload_file_to_gs_uri(file: File, address: &str, name: &str) -> Result<St
     ))
 }
 
+/// Uploads `data` to the specified gs `address` with the name `name`. Returns the gs uri for the
+/// created GCS object
+///
+/// Uses the `storage_hub` to place a GET request to the object at `address` using the Google Cloud
+/// Storage JSON API, specifically to retrieve the file contents as a String
+pub fn upload_text_to_gs_uri(data: &str, address: &str, name: &str) -> Result<String, Error> {
```

Comment:
What would be the best way to test this method?  This seems important enough that there should be some testing for it.